documentation/asciidoc/computers/raspberry-pi/otp-bits.adoc

```diff
@@ -1,16 +1,19 @@
 == OTP Register and Bit Definitions
 
-All SoCs used by the Raspberry Pi range have a inbuilt One-Time Programmable (OTP) memory block.
+All SoCs used by the Raspberry Pi range have a inbuilt One-Time Programmable (OTP) memory block. A few locations have factory-programmed data.
 
-It is 66 32-bit values long, although only a few locations have factory-programmed data.
+OTP memory size:
+
+* 2711: 66 32-bit values
```

Comment:
Does "2711" here actually mean "BCM2835, BCM2836, BMC2837 & BCM2711" ?

Reply:
Good point - I actually mean "not 2712"

Reply:
I've pushed a commit to change these references to "non-BCM2712 devices"